ui/src/components/Note.js

```diff
@@ -0,0 +1,169 @@
+    defaultMessage: 'I am this person',
+    description: ('An option for "Status" field of a note about a specific '
+        + 'person, indicating that the author of the note is the person in '
+        + 'question.'),
+  },
+  believedAlive: {
+    id: 'Note.believedAlive',
+    defaultMessage: 'I have received information that this person is alive',
+    description: ('An option for "Status" field of a note about a specific '
+        + 'person, indicating that the author of the note has received '
+        + 'information that the person in question is alive.'),
+  },
+  believedMissing: {
+    id: 'Note.believedMissing',
+    defaultMessage: 'I have reason to think this person is missing',
+    description: ('An option for "Status" field of a note about a specific '
+        + 'person, indicating that the author of the note has reason to '
+        + 'believe that the person in question is still missing.'),
+  },
+  believedDead: {
+    id: 'Note.believedDead',
+    defaultMessage: 'I have received information that this person is dead',
+    description: ('An option for "Status" field of a note about a specific '
+        + 'person, indicating that author of the note has received '
+        + 'information that the person in question is dead.'),
+  },
+  authorMadeContact: {
+    id: 'Note.authorMadeContact',
+    defaultMessage: 'This person has been in contact with someone',
+    description: ('A message which can be attached to a note about a specific '
+        + 'person.'),
+  },
+  lastKnownLocation: {
+    id: 'Note.lastKnownLocation',
+    defaultMessage: 'Last known location',
+    description: ('A label for a field of a note about a specific person, '
+        + 'containing the last known location of the person.'),
+  },
+  map: {
+    id: 'Note.map',
+    defaultMessage: 'Map',
+    description: ('A label for a button which shows a map.'),
+  },
+  postedBy: {
+    id: 'Note.postedBy',
+    defaultMessage: 'Posted by {authorName}',
+    description: ('A title of a note about a speicifc person, indicating the '
+        + 'name of the author of the note.'),
+  },
+  status: {
+    id: 'Note.status',
+    defaultMessage: 'Status',
+    description: ('A label of a field in a note about a speicifc person, '
+        + 'indicating the status of the person sought or found e.g., the '
+        + 'person is alive, missing or dead.'),
+  },
+});
+
+const STATUS_MESSAGES = {
+  '': MESSAGES.unspecified,
+  'information_sought': MESSAGES.informationSought,
+  'is_note_author': MESSAGES.isNoteAuthor,
+  'believed_alive': MESSAGES.believedAlive,
+  'believed_missing': MESSAGES.believedMissing,
+  'believed_dead': MESSAGES.believedDead,
+};
+
+/**
+ * A component which shows a single note record.
+ *
+ * Usage: <Note note={note} />
+ */
+class Note extends Component {
+  renderStatus() {
+    return <FormattedMessage {...STATUS_MESSAGES[this.props.note.status]} />;
+  }
+
+  render() {
+    return (
+      <div className='note-card'>
```

Comment:
Is there a reason not to use the MDC Card component here? Since each note has its own card it seems like it'd be a good candidate for the Card component. IIRC, you didn't use MDC Cards for the person record itself to share code with the results list, but I can't think of anything this code would be shared with.